developer_manual/core/theming.rst

```diff
@@ -43,7 +43,11 @@ All themes, whether copied or new, must meet two key criteria, these are:
    encourage you to store them in `a custom app directory`_.
 
 .. note:: 
-   When copying the example theme supplied with ownCloud in ``apps/theme-example``, make sure that you delete ``appinfo/signature.json``. If you don’t, when you make changes, an integrity warning will be triggered, as the original theme is `signed`_. To get rid of the warning you have to remove (or move) ``appinfo/signature.json`` and then trigger a rescan, in the admin page, which you can see in the screenshot below.
+   An example theme is available to base new themes on. 
+   You can find it at https://github.com/owncloud/theme-example.
+   When using the example theme, make sure that you delete ``appinfo/signature.json``. 
```

Comment:
There is no signature in the repo - there will never be. This instruction can be removed

Reply:
Must have been old. Will remove.